src/dsymbol.d

```diff
@@ -1207,6 +1207,9 @@ private:
     Dsymbols* importedScopes;
     PROTKIND* prots;            // array of PROTKIND, one for each import
 
+    import std.bitmanip : BitArray;
+    BitArray accessiblePackages;// whitelist of accessible (imported) packages
```

Comment:
> The dependency goes from 2.067 to dmd master, not from the last release.

The latest dmd compiler_s_, whatever version ldc and gdc are currently at.
Without phobos dmd can't become an idiomatic D program, we won't have the healthy learning effect of using D in a big program, productivity gains remain small (in the order of all those tiny foreach changes), and we won't attract new contributions.
We know very well from druntime, that D w/o phobos makes it a lot harder to implement features, so we end up w/ suboptimal choices and a lot of copied code.

The fact that you're asking me to implement a fundamental data structure for this PR already illustrates the problem.

> I suspect use of phobos, especially in the C++-boundary API, is going to make my work on automatic generation of headers much more difficult.

Why C++ headers, the only reason we still need C++ in the frontend and glue layer is that the other compilers haven't yet switched to D.